src/patternfly/components/Form/form.scss

```diff
@@ -24,13 +24,14 @@ $pf-v5-c-form--m-horizontal--breakpoint-map: build-breakpoint-map("sm", "md", "l
   }
 }
 
-.#{$form} {
-  --#{$form}--GridGap: var(--#{$pf-global}--gutter--md);
+:root {
```

Comment:
This and the other files are missing the component selector, `:root, .#{$form}` etc